### DISCUSSION TITLE
method NetworkInterfaces_CreateOrUpdate is not awaitable

### DISCUSSION THREAD

**Discussion by vwake7** [stackql/stackql#412] [Q&A] [answered]
@jeffreyaven I tried to Create NIC using
```
Insert /*+ AWAIT  */ into azure.network.interfaces
```
But I got the error
```
{'message': 'method NetworkInterfaces_CreateOrUpdate is not awaitable\n', 'rowsaffected': 1}
```
So just trying to understand if query hints are not supported in Azure. I am trying to create dependent resources before creating vm using pystackql. So I want the subsequent queries to wait before the current query is executed.

**Comment by jeffreyaven** [comment, marked as answer]
hey @vwake7 query hints haven't been implemented in Azure as yet, to manage dependencies you should use [__`stackql-deploy`__](https://pypi.org/project/stackql-deploy/), you define post deploy tests which will retry and block until passed before processing the next resource
